doc/user-guide/cli.md

```diff
@@ -47,6 +47,12 @@ add the `--watch` flag.
 ploomber-cloud deploy --watch
 ```
 
+You can also use the `--watch-incremental` flag to view only the newly added logs with every polling.
```

Comment:
I think this description is confusing it says "only the newly added logs", does that mean that `--watch` will print all the logs on every update? (I don't think that's the case)

it's unclear wh I'd use one option over the other one

Reply:
I think the gist of it is:

use `--watch` if deploying from your terminal as it'll clear your screen
use `--watch-incremental` if storing the deployment output into a file (e.g. github actions) as it won't attempt to clear the screen and only print new logs and status updates

Reply:
Sorry for missing the previous comment.

[fixed](https://github.com/ploomber/doc/pull/211/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R54-R55)